roles/nvidia-dgx-firmware/tasks/main.yml

```diff
@@ -30,6 +30,22 @@
   - include: load-image.yml
     when: load_firmware
 
+  # Shut down services and unload drivers
```

Comment:
Should we add a note that this is to be extra safe or work around current firmware container limitations here?